src/rewritable_units/element.rs

```diff
@@ -46,22 +46,22 @@ pub struct Element<'r, 't> {
     end_tag_mutations: Option<Mutations>,
     modified_end_tag_name: Option<Bytes<'static>>,
     end_tag_handler: Option<EndTagHandler<'static>>,
-    can_have_content: bool,
+    el_can_have_content: bool,
```

Comment:
Given that this is a `Element` the `el_` doesn't add much.  I think the original name is better.

Reply:
I agree, but the field and the function can't have the same name.  Maybe the field could keep it's name and we just make it `pub` instead?

Reply:
> the field and the function can't have the same name

It can.  Give it a try :P

Reply:
Weird, I could have sworn I tried that first and the compiler didn't like it.  Fixed.